crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -93,24 +96,27 @@ where
         metrics,
         indexed_checkpoint_sender,
         package_buffer: IndexingPackageBuffer::start(package_tx),
+        _phantom: Default::default(),
```

Comment:
what's this and how is this used?

Reply:
This is because we added `_phantom: PhantomData<T>,` in the struct definition, we need to remember the connection type so later we can return the type in the connection pool. Check below function: `fn pg_blocking_cp(&self) -> Result<ConnectionPool<T>, IndexerError> {`